### DISCUSSION TITLE
[일지] 6월 9일 (수) 오전 회의록

### DISCUSSION THREAD

**Discussion by olenmg** [bcaitech1/p4-opt-5-vibrhanium-#67] [일지]
### ==노래 당첨==
- 추천곡 : 

### 어제 배정된 일 점검
`재경`: Deep compression 논문 - quantization
`종헌`: quantization 어제/오늘 강의 다 정리해와서 발표
`아름`: squeezenet with deep-compression 다른 [frame work](https://github.com/songhan/SqueezeNet-Deep-Compression)로 되어있는거 pytorch 로 변환 도전...!!! Deep compression 논문 - huffman coding
`건우`: pretrained 모델에 pruning 적용 방법 구글링 
`재영`: efficientNet 구현! 
`명환`: Deep compression 논문 - prunning 

### 대회
- shuffleNet image size 32 => 80만 (MACs)
    - 지금까지 학습한 것 제출해보기
    - 학습한거 불러와서 더 오래 학습시켜보기 
    - sample data로 학습시켜보기
    - (참고)image size 56으로 하면 250~260만 MACs

### 할 일
`재경`: knowledge distilation ? quantization 더 파보기?
`종헌`: 오후 발표 준비, 오늘 강의에 어떤 내용이 나오는지까지 정리해보겠습니다 
`아름`: shuffleNet 학습 이어서 시키기, 강의 듣기
`건우`: ༼ つ ◕_◕ ༽つ pruning pytorch 튜토리얼 진행, 실제 모델 적용
`재영`: 다른 모델 탐색, 한거 정리
`명환`: AutoML에서 pruning 적용? 혹은 안된다면 기존 모델에서 pruning 적용
